oauth2_provider/models.py

```diff
@@ -234,8 +234,9 @@ def clean(self):
             ):
                 raise ValidationError(_("You cannot use HS256 with public grants or clients"))
 
+    # TODO: I removed usage of this in templates. so it can label as deprecated.`
```

Comment:
What's this new TODO comment about?

Reply:
PS: Also please close #1445 since you've moved the change over to this PR.

Reply:
Method `get_absolute_url` one time used in one template for getting url of a page. I used `{% url %}` tag bug i suggest to change it to deprecated instead of removing it. It is unusual to remove or change behavior of a method suddenly.

Reply:
```suggestion
```
Suggest you remove the comment and just leave the `get_absolute_url` in case someone may be using it elsewhere (e.g. in their own template or other code based on the application model class). It's probably not worth deprecating it and then keeping track that it has to get removed in a future release.